api/routers/edr.py

```diff
@@ -73,7 +74,9 @@ async def get_data_location_id(
     get_obs_request = dstore.GetObsRequest(
         filter=dict(
             platform=dstore.Strings(values=[location_id]),
-            instrument=dstore.Strings(values=list(map(str.strip, parameter_name.split(",")))),
+            instrument=dstore.Strings(
+                values=list(map(str.strip, parameter_name.split(","))) if parameter_name else "*"
+            ),
```

Comment:
Better to leave the whole `instrument` field out if you don't want to filter on it. I think the code will be slightly clearer, and it will more efficient in the store.

Also for the other endpoint.

Reply:
Now leaves `instrument` out when not needed https://github.com/EURODEO/e-soh/pull/30/commits/36e1a0e93a46b2f9a619f2f29a9eb16278107318